set.mm

```diff
@@ -29355,6 +29355,12 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
     risset $p |- ( A e. B <-> E. x e. B x = A ) $=
       ( cv wcel wceq wa wex wrex exancom df-rex dfclel 3bitr4ri ) ADZCEZNBFZGAH
       POGAHPACIBCEOPAJPACKABCLM $.
+
+    $( A definition of ` -. A e. B ` .  (Contributed by Thierry Arnoux,
+       20-Nov-2023.)  (Proof shortened by SN, 23-Jan-2024.) $)
+    nelb $p |- ( -. A e. B <-> A. x e. B x =/= A ) $=
+      ( cv wne wral wcel wn wceq wrex df-ne ralbii ralnex bitri risset xchbinxr
+      bicomi ) ADZBEZACFZBCGZHTRBIZACJZUATUBHZACFUCHSUDACRBKLUBACMNABCOPQ $.
```

Comment:
This is an example of where the current definition of df-ral introduces some maybe unwanted trickery.  Neither A nor B need to be independent of x, so what is expressed in such a situation?  It would be clearer in my eyes to replace x with a z that is disjoint from B, and A.  Or do you need this (for me) hard to understand case where x is allowed to appears in all classes?

Reply:
Actually `x` needs to be independent of `A` and `B`, at least in the original proof:
```
  ${
    $d A x $.  $d B x $.
    $( A definition of ` -. A e. B ` .  (Contributed by Thierry Arnoux,
       20-Nov-2023.) $)
    nelb $p |- ( -. A e. B <-> A. x e. B x =/= A ) $=
      ( cv wceq wn wral wcel wi wal wne df-ral df-ne ralbii wa wex dfclel alnex
      notbii imnan ancom bitr2i albii 3bitr2i 3bitr4ri ) ADZBEZFZACGUFCHZUHIZAJ
      ZUFBKZACGBCHZFZUHACLULUHACUFBMNUNUGUIOZAPZFUOFZAJUKUMUPABCQSUOARUQUJAUJUI
      UGOZFUQUIUGTURUOUIUGUASUBUCUDUE $.
  $}
```
The same conditions appear just before `risset`, and in this PR `nelb` is moved to the same frame as `risset`, so it's still the case in the shortened version.

Reply:
The distinct variable condition is actually visible just below before what becomes `nelbOLD`

Reply:
ok, yes the dv conditions are placed in front of risset.